packages/elements/src/slider/__test__/slider.test.js

```diff
@@ -464,4 +474,15 @@ describe('slider/Slider', function () {
       expect(el.stepRange).to.equal(5.5);
     });
   });
+
+  describe('Marker', function () {
+    it('Set position of marker correctly when new marker added', async function () {
```

Comment:
let's add another test case checking position of the marker when it's min and max are not 100. We should make sure that it scaling correctly.

Reply:
updated